# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/gv/docdir.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config.Unix~	Sat Jun  7 00:00:00 1997
+++ ./config.Unix	Sun Mar 19 08:05:32 2000
@@ -33,7 +33,7 @@
 XCOMM  Destination directories.
 XCOMM --------------------------------------------------------------------
       GV_BINDIR    = /usr/local/bin/
-      GV_DOCDIR    = /usr/local/doc/gv/
+      GV_DOCDIR    = /usr/local/doc/packages/gv/
       GV_LIBDIR    = /usr/local/lib/gv/
       GV_MANDIR    = /usr/local/man/man1/
 
